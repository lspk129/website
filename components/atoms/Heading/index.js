import styled from 'styled-components';

const Heading = styled.h3`
  text-align: center;
  text-transform: ${props => (props.upperCase ? 'uppercase' : '')};
`;

export default Heading;
