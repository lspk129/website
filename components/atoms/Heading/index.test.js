import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import Heading from './';

describe('Heading', () => {
  const shallowWrapper = shallow(<Heading />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<Heading />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders h3 tag', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('h3');
  });
});
