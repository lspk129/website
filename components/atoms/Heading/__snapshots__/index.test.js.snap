// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders correctly 1`] = `
<h3
  className="sc-bdVaJa csGkgJ"
/>
`;
